source/AI.cpp

```diff
@@ -2594,6 +2639,113 @@ bool AI::DoCloak(Ship &ship, Command &command)
 
 
 
+bool AI::DoRoving(Ship &ship, Command &command, unsigned int rovingShipCount)
+{	
+	const Point target = ship.GetTargetPosition();
+	const auto v = ship.MaxVelocity();
+	if(!target || MoveTo(ship, command, target, Point(), v, v))
+	{
+		Point newTarget = Angle::Random().Unit() * Random::Real() * MAX_DISTANCE_FROM_CENTER;
+		ship.SetTargetPosition(newTarget);
+	}
+	
+	const System *system = ship.GetSystem();
+	if(rovingShipCount > 9 && system)
```

Comment:
I mostly have a problem with this because the number is hardcoded and I cant find any logic behind it
Would be meat to idk define it by government with a default of 9 for instance

Reply:
That is a good idea. Perhaps even better would be being able to specify the number of ships in the fleet definition.